docs/lib-documentation.md

```diff
@@ -41,6 +40,19 @@ Gets the [Service][service] interface representing the service with the given ID
 
 The [Service][service] implementation representing the service.
 
+### getServiceContext(ServiceID serviceId, [ContainerConfigFactory\<S\>][containerconfigfactory] configFactory)
+Gets relevant information about a service (identified by the given service ID) that is running in the network.
+
+**Args**
+
+* `serviceId`: The ID of the service to pull the information from.
+* `configFactory`: The factory that produces the configs Kurtosis will use to start the container for the new service.
+
+**Returns**
+
+The [ServiceContext][servicecontext] representation of a service running in a Docker container.
```

Comment:
Dude the English for this entire method is really good

Reply:
thanks